bots/task/test-task

```diff
@@ -100,6 +110,19 @@ def mockServer():
     httpd.serve_forever()
     os._exit(1)
 
+def mock_execute(*args):
+    if args[0] != 'git':
+        raise Exception("Mocking unsupported command")
+    if args[1] == "show":
+        return "Task title\n"
+    elif args[1] == "commit" and args[2] == "--amend":
+        if args[4] !=  "Task title\nCloses #1234":
+            raise Exception("Incorrect commit message")
+    elif args[1] == "push":
+        pass
```

Comment:
would be nice to validate that you get expected values. If push gets called with two different args (one with -f, one without), save `args` to `mock_execute.last_args` and assert their value in `testPullBody()`.